src/manifests_operator.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Ubuntu
+# See LICENSE file for licensing details.
+
+"""Implementation of intel-device-plugins-operator kubernetes manifests."""
+
+import logging
+import pickle
+from hashlib import md5
+from typing import Dict
+
+from ops.manifests import ConfigRegistry, ManifestLabel, Manifests, Patch
+
+log = logging.getLogger()
+
+
+class RemoveNamespace(Patch):
+    """Remove namespace from resources so they deploy to the charm's namespace."""
+
+    def __call__(self, obj):
+        """Remove namespace from resources."""
+        obj.metadata.namespace = None
+
+
+class IntelDevicePluginsK8SOperatorManifests(Manifests):
```

Comment:
Again -- looks like you're not really making any changes to the yaml from upstream.  Again -- this is a very easy deployment